build/dangerjs/checkChangelog.ts

```diff
@@ -20,8 +20,8 @@ const hasAddedLinesAfterVersionInChangelog = async (danger): Promise<boolean> =>
 }
 
 const getMalformedChangelogEntries = async (danger): Promise<string[]> => {
-  // +- description @githubname ([#DDDD](https://github.com/stardust-ui/react/pull/DDDD))
-  const validEntry = /^\+- .*@\S+ \(\[#\d+]\(https:\/\/github\.com\/stardust-ui\/react\/pull\/\d+\)\)$/
+  // +- description @githubname ([#DDDD](https://github.com/ANYTHING/pull/DDDD))
```

Comment:
Just curious, why not white-listing: `microsoft/fluent-ui-react` and `stardust-ui/react` as only available options?

Reply:
Because I wanted to make it rename-agnostic 😛 
And to keep the regex sane.

OK, changed to `stardust|fluent`